canvas_modules/common-canvas/__tests__/common-properties/common-properties-test.js

```diff
@@ -134,186 +134,214 @@ const callbacks = {
 };
 
 describe("CommonProperties renders correctly", () => {
+	const spy = sinon.spy();
+	const SpyWrapper = (props) => {
+		spy(props); // capture all props passed
+		return <CommonProperties {...props} />;
+	};
 
 	it("all required props should have been defined", () => {
-		const wrapper = createCommonProperties("Editing");
-		expect(wrapper.prop("propertiesInfo")).to.equal(propertiesInfo);
-		expect(wrapper.prop("callbacks")).to.equal(callbacks);
-		expect(wrapper.prop("propertiesConfig").containerType).to.equal("Editing");
+		const propertiesConfig = { containerType: "Editing" };
+		render(
+			<IntlProvider key="IntlProvider2" locale={locale} messages={{}}>
+				<SpyWrapper
+					propertiesInfo={propertiesInfo}
+					propertiesConfig={propertiesConfig}
+					callbacks={callbacks}
+				/>
+			</IntlProvider>
+		);
+		expect(spy.calledOnce).to.be.true;
+		const props = spy.firstCall.args[0];
+		expect(props.propertiesInfo).to.equal(propertiesInfo);
+		expect(props.callbacks).to.equal(callbacks);
+		expect(props.propertiesConfig.containerType).to.equal("Editing");
 	});
 
 	it("should render one <PropertiesDialog/> component", () => {
-		const wrapper = createCommonProperties("Modal");
-		expect(wrapper.find(PropertiesDialog)).to.have.length(1);
+		createCommonProperties("Modal");
+
+		// Since PropertiesModal is rendered using a React Portal, it is mounted outside the main container.
+		const propertiesModal = document.body.querySelector(".properties-modal");
+		expect(propertiesModal).to.exist;
```

Comment:
In several test cases,` .to.have.length(1)` is replaced with `to.exist`. Does RTL have a way to check length?